docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
-3. Pull the `apprenda/pause` image from `https://hub.docker.com/r/apprenda/pause`. 
-4. RRAS (Routing) Windows feature enabled.
-5. Install a VMSwitch of type `Internal`, by running `New-VMSwitch -Name KubeProxySwitch -SwitchType Internal` command in *PowerShell* window. This will create a new Network Interface with name `vEthernet (KubeProxySwitch)`. This interface will be used by kube-proxy to add Service IPs.
 
 **Linux Host Setup**
 
 1. Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using. 
-2. CNI network plugin installed.
-
-### Component Setup
-
-Requirements
-
-* Git
-* Go 1.7.1+
-* make (if using Linux or MacOS)
-* Important notes and other dependencies are listed [here](https://git.k8s.io/community/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment)
-
-**kubelet**
-
-To build the *kubelet*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kubelet*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet`
-   2. Windows: `go build cmd/kubelet/kubelet.go`
-
-**kube-proxy**
-
-To build *kube-proxy*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kube-proxy*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy`
-   2. Windows: `go build cmd/kube-proxy/proxy.go`
+2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+3. [Optional] CNI network plugin installed.
 
-### Route Setup
-The below example setup assumes one Linux and two Windows Server 2016 nodes and a cluster CIDR 192.168.0.0/16
-
-| Hostname | Routable IP address | Pod CIDR |
-| --- | --- | --- |
-| Lin01 | `<IP of Lin01 host>` | 192.168.0.0/24 |
-| Win01 | `<IP of Win01 host>` | 192.168.1.0/24 |
-| Win02 | `<IP of Win02 host>` | 192.168.2.0/24 |
-
-**Lin01**
-
-```
-ip route add 192.168.1.0/24 via <IP of Win01 host>
-ip route add 192.168.2.0/24 via <IP of Win02 host>
-```
+**Windows Host Setup**
 
-**Win01**
+1. Windows Server container host running Windows Server version 1709 and Docker v17.06 or later. Follow the setup instructions outlined by this help topic: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server.
+2. Build or download kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions found [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+3. Copy Node spec file (config) from Linux master node with X.509 keys
+4. Create HNS Network
+5. Ensure correct CNI network config
+5. Start kubelet.exe using this script [start-kubelet.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/start-kubelet.ps1)
+6. Start kube-proxy using this script[start-kubeproxy.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/start-kubeproxy.ps1)
```

Comment:
nit: space after "script"

Reply:
If you're starting using the script, you don't need steps 4 & 5. I like that it basically outlines what the script will do so it's not a black box, but maybe it should just be inline with the kubelet step.

Reply:
addressed in https://github.com/JMesser81/kubernetes.github.io/pull/1